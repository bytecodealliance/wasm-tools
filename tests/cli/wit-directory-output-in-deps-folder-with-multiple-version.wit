// RUN[gen]: component wit % --out-dir %tmpdir
// RUN[read]: component wit %tmpdir

package foo:root;
package a:b@0.2.0 {
  interface foo {}
}

package a:b {
  interface foo {}
}

package a:c {
  world a {
    import a:b/foo@0.2.0;
    import a:b/foo;
  }
}
