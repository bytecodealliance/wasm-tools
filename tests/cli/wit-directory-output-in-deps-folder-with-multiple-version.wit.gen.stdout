Writing: %tmpdir/deps/b.wit
Writing: %tmpdir/deps/b@0.2.0.wit
Writing: %tmpdir/deps/c.wit
Writing: %tmpdir/root.wit
